content/en/docs/reference/glossary/cloud-controller-manager.md

```diff
@@ -12,9 +12,9 @@ tags:
 - operation
 ---
  A Kubernetes {{< glossary_tooltip text="control plane" term_id="control-plane" >}} component
-that embeds cloud-specific control logic. The cloud controller manager lets you link your
-cluster into your cloud provider's API, and separates out the components that interact
-with that cloud platform from components that only interact with your cluster.
+that embeds cloud-specific control logic. The [cloud controller manager](docs/concepts/architecture/cloud-controller/)
```

Comment:
This currently leads to a 404. Modifying it as below would lead it to the correct page.
```suggestion
that embeds cloud-specific control logic. The [cloud controller manager](/docs/concepts/architecture/cloud-controller/)
```